engine/runtime/src/main/scala/org/enso/interpreter/runtime/DefaultPackageRepository.scala

```diff
@@ -578,15 +578,15 @@ private class DefaultPackageRepository(
     val cache = ensurePackageIsLoaded(libraryName).toOption.flatMap { _ =>
       if (!loadedLibraryBindings.contains(libraryName)) {
         loadedPackages.get(libraryName).flatten.foreach(loadDependencies(_))
-        context
+        val cachedBindingOption = context
           .asInstanceOf[TruffleCompilerContext]
           .getSerializationManager()
           .deserializeLibraryBindings(libraryName)
-          .foreach(cache => loadedLibraryBindings.addOne((libraryName, cache)))
+        loadedLibraryBindings.addOne((libraryName, cachedBindingOption))
```

Comment:
When I tried to move away a `.bindings` file with:
```bash
enso$ mv ./built-distribution/enso-engine-0.0.0-dev-linux-amd64/enso-0.0.0-dev/lib/Standard/Base/0.0.0-dev/.enso/cache/bindings/0.0.0-dev/Standard/Base.bindings x
```
and reading the cache generated `IOException` I noticed that the load is attempted many times. Avoiding it by storing even `None` into the cache. CCing @hubertp.

Reply:
I don't think it is cosmetic. I need to run my code even on `None`. `None.flatMap` is no-op (isn't it?) - e.g. it wouldn't record the `None` result into the map.